<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalculatorUI</class>
 <widget class="QWidget" name="CalculatorUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>계산기</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: black;
    color: white;
    font-family: 'Helvetica Neue', 'Arial';
}

QLineEdit#display {
    font-size: 90px;
    font-weight: 300;
    border: none;
    qproperty-alignment: 'AlignRight';
    padding: 0 10px;
}

QLabel#radLabel {
    color: #999;
    font-size: 20px;
    font-weight: bold;
    padding-left: 15px;
}

QPushButton {
    font-size: 28px;
    font-weight: 500;
    border: none;
    min-width: 78px;
    max-width: 78px;
    min-height: 78px;
    max-height: 78px;
    border-radius: 39px;
}

QPushButton:pressed {
    background-color: #555555;
}

QPushButton[class=&quot;light&quot;] {
    background-color: #A5A5A5;
    color: black;
}
QPushButton[class=&quot;light&quot;]:pressed {
    background-color: #CCCCCC;
}

QPushButton[class=&quot;orange&quot;] {
    background-color: #FF9500;
    color: white;
}
QPushButton[class=&quot;orange&quot;]:pressed {
    background-color: #FFAE3C;
}

QPushButton[class=&quot;dark&quot;] {
    background-color: #333333;
    color: white;
}
QPushButton[class=&quot;dark&quot;]:pressed {
    background-color: #555555;
}

QPushButton#zeroButton {
    min-width: 168px;
    max-width: 168px;
    text-align: left;
    padding-left: 32px;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="2,0,5">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <widget class="QLineEdit" name="display">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>120</height>
      </size>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="radLabel">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>Rad</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="buttons_layout">
     <property name="spacing">
      <number>12</number>
     </property>

     <item row="0" column="0"><widget class="QPushButton" name="btn_lparen"><property name="text"><string>(</string></property><property name="class" stdset="0"><string>dark</string></property></widget></item>
     <item row="0" column="1"><widget class="QPushButton" name="btn_rparen"><property name="text"><string>)</string></property><property name="class" stdset="0"><string>dark</string></property></widget></item>
     <item row="0" column="2"><widget class="QPushButton" name="btn_mc"><property name="text"><string>mc</string></property><property name="class" stdset="0"><string>dark</string></property></widget></item>
     <item row="0" column="3"><widget class="QPushButton" name="btn_mplus"><property name="text"><string>m+</string></property><property name="class" stdset="0"><string>dark</string></property></widget></item>
     <item row="0" column="4"><widget class="QPushButton" name="btn_mminus"><property name="text"><string>m-</string></property><property name="class" stdset="0"><string>dark</string></property></widget></item>
     <item row="0" column="5"><widget class="QPushButton" name="btn_mr"><property name="text"><string>mr</string></property><property name="class" stdset="0"><string>dark</string></property></widget></item>
     <item row="0" column="6"><widget class="QPushButton" name="btn_ac"><property name="text"><string>AC</string></property><property name="class" stdset="0"><string>light</string></property></widget></item>
     <item row="0" column="7"><widget class="QPushButton" name="btn_plusminus"><property name="text"><string>±</string></property><property name="class" stdset="0"><string>light</string></property></widget></item>
     <item row="0" column="8"><widget class="QPushButton" name="btn_percent"><property name="text"><string>%</string></property><property name="class" stdset="0"><string>light</string></property></widget></item>
     <item row="0" column="9"><widget class="QPushButton" name="btn_divide"><property name="text"><string>÷</string></property><property name="class" stdset="0"><string>orange</string></property></widget></item>

     <item row="1" column="0"><widget class="QPushButton" name="btn_2nd"><property name="text"><string>2nd</string></property><property name="class" stdset="0"><string>dark</string></property></widget></item>
     <item row="1" column="1"><widget class="QPushButton" name="btn_sqr"><property name="text"><string>x^2</string></property><property name="class" stdset="0"><string>dark</string></property></widget></item>
     <item row="1" column="2"><widget class="QPushButton" name="btn_cub"><property name="text"><string>x^3</string></property><property name="class" stdset="0"><string>dark</string></property></widget></item>
     <item row="1" column="3"><widget class="QPushButton" name="btn_pow_y"><property name="text"><string>x^y</string></property><property name="class" stdset="0"><string>dark</string></property></widget></item>
     <item row="1" column="4"><widget class="QPushButton" name="btn_e_pow_x"><property name="text"><string>e^x</string></property><property name="class" stdset="0"><string>dark</string></property></widget></item>
     <item row="1" column="5"><widget class="QPushButton" name="btn_10_pow_x"><property name="text"><string>10^x</string></property><property name="class" stdset="0"><string>dark</string></property></widget></item>
     <item row="1" column="6"><widget class="QPushButton" name="btn_7"><property name="text"><string>7</string></property><property name="class" stdset="0"><string>dark</string></property></widget></item>
     <item row="1" column="7"><widget class="QPushButton" name="btn_8"><property name="text"><string>8</string></property><property name="class" stdset="0"><string>dark</string></property></widget></item>
     <item row="1" column="8"><widget class="QPushButton" name="btn_9"><property name="text"><string>9</string></property><property name="class" stdset="0"><string>dark</string></property></widget></item>
     <item row="1" column="9"><widget class="QPushButton" name="btn_multiply"><property name="text"><string>×</string></property><property name="class" stdset="0"><string>orange</string></property></widget></item>

     <item row="2" column="0"><widget class="QPushButton" name="btn_reciprocal"><property name="text"><string>¹/ₓ</string></property><property name="class" stdset="0"><string>dark</string></property></widget></item>
     <item row="2" column="1"><widget class="QPushButton" name="btn_sqrt"><property name="text"><string>√x</string></property><property name="class" stdset="0"><string>dark</string></property></widget></item>
     <item row="2" column="2"><widget class="QPushButton" name="btn_cbrt"><property name="text"><string>∛x</string></property><property name="class" stdset="0"><string>dark</string></property></widget></item>
     <item row="2" column="3"><widget class="QPushButton" name="btn_yroot"><property name="text"><string>ʸ√x</string></property><property name="class" stdset="0"><string>dark</string></property></widget></item>
     <item row="2" column="4"><widget class="QPushButton" name="btn_ln"><property name="text"><string>ln</string></property><property name="class" stdset="0"><string>dark</string></property></widget></item>
     <item row="2" column="5"><widget class="QPushButton" name="btn_log10"><property name="text"><string>log10</string></property><property name="class" stdset="0"><string>dark</string></property></widget></item>
     <item row="2" column="6"><widget class="QPushButton" name="btn_4"><property name="text"><string>4</string></property><property name="class" stdset="0"><string>dark</string></property></widget></item>
     <item row="2" column="7"><widget class="QPushButton" name="btn_5"><property name="text"><string>5</string></property><property name="class" stdset="0"><string>dark</string></property></widget></item>
     <item row="2" column="8"><widget class="QPushButton" name="btn_6"><property name="text"><string>6</string></property><property name="class" stdset="0"><string>dark</string></property></widget></item>
     <item row="2" column="9"><widget class="QPushButton" name="btn_subtract"><property name="text"><string>−</string></property><property name="class" stdset="0"><string>orange</string></property></widget></item>

     <item row="3" column="0"><widget class="QPushButton" name="btn_factorial"><property name="text"><string>x!</string></property><property name="class" stdset="0"><string>dark</string></property></widget></item>
     <item row="3" column="1"><widget class="QPushButton" name="btn_sin"><property name="text"><string>sin</string></property><property name="class" stdset="0"><string>dark</string></property></widget></item>
     <item row="3" column="2"><widget class="QPushButton" name="btn_cos"><property name="text"><string>cos</string></property><property name="class" stdset="0"><string>dark</string></property></widget></item>
     <item row="3" column="3"><widget class="QPushButton" name="btn_tan"><property name="text"><string>tan</string></property><property name="class" stdset="0"><string>dark</string></property></widget></item>
     <item row="3" column="4"><widget class="QPushButton" name="btn_e"><property name="text"><string>e</string></property><property name="class" stdset="0"><string>dark</string></property></widget></item>
     <item row="3" column="5"><widget class="QPushButton" name="btn_ee"><property name="text"><string>EE</string></property><property name="class" stdset="0"><string>dark</string></property></widget></item>
     <item row="3" column="6"><widget class="QPushButton" name="btn_1"><property name="text"><string>1</string></property><property name="class" stdset="0"><string>dark</string></property></widget></item>
     <item row="3" column="7"><widget class="QPushButton" name="btn_2"><property name="text"><string>2</string></property><property name="class" stdset="0"><string>dark</string></property></widget></item>
     <item row="3" column="8"><widget class="QPushButton" name="btn_3"><property name="text"><string>3</string></property><property name="class" stdset="0"><string>dark</string></property></widget></item>
     <item row="3" column="9"><widget class="QPushButton" name="btn_add"><property name="text"><string>+</string></property><property name="class" stdset="0"><string>orange</string></property></widget></item>

     <item row="4" column="0"><widget class="QPushButton" name="btn_deg"><property name="text"><string>Deg</string></property><property name="class" stdset="0"><string>dark</string></property></widget></item>
     <item row="4" column="1"><widget class="QPushButton" name="btn_sinh"><property name="text"><string>sinh</string></property><property name="class" stdset="0"><string>dark</string></property></widget></item>
     <item row="4" column="2"><widget class="QPushButton" name="btn_cosh"><property name="text"><string>cosh</string></property><property name="class" stdset="0"><string>dark</string></property></widget></item>
     <item row="4" column="3"><widget class="QPushButton" name="btn_tanh"><property name="text"><string>tanh</string></property><property name="class" stdset="0"><string>dark</string></property></widget></item>
     <item row="4" column="4"><widget class="QPushButton" name="btn_pi"><property name="text"><string>π</string></property><property name="class" stdset="0"><string>dark</string></property></widget></item>
     <item row="4" column="5"><widget class="QPushButton" name="btn_rand"><property name="text"><string>Rand</string></property><property name="class" stdset="0"><string>dark</string></property></widget></item>
     <item row="4" column="6" colspan="2"><widget class="QPushButton" name="btn_0"><property name="text"><string>0</string></property><property name="class" stdset="0"><string>dark</string></property></widget></item>
     <item row="4" column="8"><widget class="QPushButton" name="btn_decimal"><property name="text"><string>.</string></property><property name="class" stdset="0"><string>dark</string></property></widget></item>
     <item row="4" column="9"><widget class="QPushButton" name="btn_equals"><property name="text"><string>=</string></property><property name="class" stdset="0"><string>orange</string></property></widget></item>

    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>